sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,72 @@
+WITH flows AS (
+  -- last fxa_uid for each flow_id
+  SELECT
+    flow_id,
+    MIN(flow_started) AS flow_started,
+    ARRAY_AGG(fxa_uid IGNORE NULLS ORDER BY fxa_uid_timestamp DESC LIMIT 1)[
+      SAFE_OFFSET(0)
+    ] AS fxa_uid,
+  FROM
+    `moz-fx-cjms-nonprod-9a36`.cjms_bigquery.flows_v1
+  WHERE
+    -- only use the last 10 days in stage
+    submission_date >= CURRENT_DATE - 10
+  GROUP BY
+    flow_id
+),
+attributed_flows AS (
+  -- last flow_id in aic for each fxa_uid
+  SELECT
+    fxa_uid,
+    ARRAY_AGG(STRUCT(flow_id, flow_started) ORDER BY flow_started DESC LIMIT 1)[SAFE_OFFSET(0)].*,
+  FROM
+    flows
+  JOIN
+    EXTERNAL_QUERY("moz-fx-cjms-nonprod-9a36.us.cjms-sql", "SELECT flow_id FROM aic")
```

Comment:
The [architecture/flow diagram](https://www.figma.com/file/6jnLCLzclBN0uyS4nJp57d/Affiliate-Marketing-(CJ)-Architecture-%2F-Flow?node-id=0%3A1) seems to indicate some data may be removed from the CJMS database:
> Has the time frame for affiliate credit passed?
> Yes:  CJMS removes the row from the unique id table

Since we're rebuilding this whole table every time, are we sure that the `aic` table will retain all the historical data we need to do that accurately?

Reply:
this table is not expected to contain historical results, only current results based on the `aic` table.